site/docs/samples/dashboard-sample.md

```diff
@@ -65,7 +61,7 @@ This enables easy [cleanup](#cleanup) once you're done experimenting with the sa
 ## Backend data server
 
 The backend data server is responsible for reading or writing data to a database or any other resource. In this 
-example the backend data server is a simple server that returns the mock data for the dashboard.    
+example the backend data server is a simple server that returns the mock container data.    
```

Comment:
the new helm (3.7) doesn't support `helm chart` commands, the code snippet below should be updated